README.md

```diff
@@ -1,38 +1,56 @@
 # discord-py-slash-command
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/224bdbe58f8f43f28a093a33a7546456)](https://app.codacy.com/gh/eunwoo1104/discord-py-slash-command?utm_source=github.com&utm_medium=referral&utm_content=eunwoo1104/discord-py-slash-command&utm_campaign=Badge_Grade_Settings)
+<a href="https://discord.gg/KkgMBVuEkx"> <img alt="Discord" src="https://img.shields.io/discord/789032594456576001" /> </a>
 
-Simple Discord Slash Commands extension for [discord.py](https://github.com/Rapptz/discord.py).
+A simplified Discord Slash Command extension for [discord.py](https://github.com/Rapptz/discord.py).
 
-## Example
-Normal usage:
+[**Discussions**](https://github.com/eunwoo1104/discord-py-slash-command/discussions).
+
+## About
```

Comment:
Um... Do we actually need this? It is cool, but seems kinda off-topic.

Reply:
Wait just in case I meant `## About` section.

Reply:
I think users should be able to have an idea about what it specifically does, and may need an understanding of the background of how slash commands are. It gives some insight into how it may take some time for a more "official" build of this library will come to be and emphasizes how new this all is.

Reply:
I can always remove it, however, if you wish.

Reply:
Seems legit. Before accepting this though, I'll make more requests.